### PR TITLE
Add scholarship page

### PR DESCRIPTION
**Pull request by carolynvs**
Preview available at https://deploy-preview-17--womenwhogo.netlify.com/scholarships.html

### CODE REVIEW

File: womenwhogo.org/scholarships.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    <hr>
+    <br>
+    <br>
+    <div class="container links">
+      <h2>Eligibility</h2>
+        <p>You must meet at least one of the following requirements:</p>
+        <ul>
+          <li>Identify as a woman</li>
+          <li>Identify as a person of color</li>
+          <li>Identify as LGBTQ+ or non-binary</li>
```
Comment (meyskens) [inline womenwhogo.org/scholarships.html]: Can we move the non-binary to the woman part? Feels weird to be sitting here

File: womenwhogo.org/scholarships.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      <p>
+      <div style="margin-bottom: 3em;">
+        <h2 style="text-align: center">
+          <a href="https://docs.google.com/forms/d/1CukfD-XPLJ87pzz1qmKCuoN6mSKSpVgvu6lydCzFNn0">GopherCon US 2018 Application Form</a>
+        </h2>
+      </div>
+    </div>
+
+    <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```
Comment (meyskens) [inline womenwhogo.org/scholarships.html]: i wonder who owns this GA tag....

File: womenwhogo.org/scholarships.html (hunk 1 above)
Comment (carolynvs) [reply womenwhogo.org/scholarships.html]: Will do 👍

File: womenwhogo.org/scholarships.html (hunk 2 above)
Comment (carolynvs) [reply womenwhogo.org/scholarships.html]: I am guessing Sarah?

Comment (meyskens) [reply womenwhogo.org/scholarships.html]: If so we should be able to log in to it. Will look into out GSuite later to see if it is there

File: womenwhogo.org/scholarships.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    <hr>
+    <br>
+    <br>
+    <div class="container links">
+      <h2>Eligibility</h2>
+        <p>You must meet at least one of the following requirements:</p>
+        <ul>
+          <li>Identify as a woman or non-binary</li>
+          <li>Identify as a person of color</li>
+          <li>Identify as LGBTQ+</li>
```
Comment (meyskens) [inline womenwhogo.org/scholarships.html]: thinking of making it `LGBTQIA+` but I could be biased on this since I am part of the `A`

Comment (carolynvs) [reply womenwhogo.org/scholarships.html]: Can we fix that on the form too? I was copying what we used there. I'll fix on the page for now.

**Comment by meyskens** [comment]
@carolynvs link for iceland: https://docs.google.com/forms/d/e/1FAIpQLScocnaSbwwLGeZLdnXuUtqlXrd4sSDX_w6Rk-ecBcnrg69ozQ/formResponse

**Pull request merged**